modules/vpc/README.md

```diff
@@ -56,7 +57,8 @@ No modules.
 | Name | Description |
 |------|-------------|
 | <a name="output_nap_public_id"></a> [nap\_public\_id](#output\_nap\_public\_id) | The public IP associated with the VPC's NAT |
-| <a name="output_private_subnet"></a> [private\_subnet](#output\_private\_subnet) | The private subnet of the created VPC |
-| <a name="output_public_subnet"></a> [public\_subnet](#output\_public\_subnet) | The public subnet of the created VPC |
+| <a name="output_private_subnets"></a> [private\_subnets](#output\_private\_subnets) | The private subnets of the created VPC |
+| <a name="output_public_subnets"></a> [public\_subnets](#output\_public\_subnets) | The public subnets of the created VPC |
+| <a name="output_routing_ready"></a> [routing\_ready](#output\_routing\_ready) | Allows to depends on completion of routing resources |
```

Comment:
@Bit-Doctor Can you clarify what `routing\_ready` does? Is it a setting that you can enable to require that routing resources be complete before another process begins?

Reply:
It's not a setting but something created once "routing resources" are complete.
It allows to wait for it to continue to the next step. Basically it is a flag raised when it ok to proceed.
In terraform this would be used like so `depends_on = [routing_ready]`

Happy to get your input on how to formulate it better :D

Reply:
Got it, thanks! My suggestion would be to change it to something along the lines of `Indicates whether routing resources have been successfully provisioned`.